tests/integration/test_object.py

```diff
@@ -61,15 +67,17 @@ async def crud_works(schema, filename):
                 LOG.debug(f"Checking that XML object {accession_id[0]} was deleted")
                 assert resp.status == 404, f"HTTP Status code error, got {resp.status}"
 
-            async with client_logged_in.get(f"{submissions_url}/{submission_fega}") as resp:
-                LOG.debug(f"Checking that object {accession_id[0]} was deleted from submission {submission_fega}")
+            async with client_logged_in.get(f"{submissions_url}/{submission_id}") as resp:
+                LOG.debug(f"Checking that object {accession_id[0]} was deleted from submission {submission_id}")
                 res = await resp.json()
                 expected_true = not any(d["accessionId"] == accession_id[0] for d in res["metadataObjects"])
                 assert expected_true, f"object {accession_id[0]} still exists"
 
         gather_items = []
         for schema, filename in test_fega_xml_files:
-            gather_items.append(crud_works(schema, filename))
+            gather_items.append(crud_works(schema, filename, submission_fega))
+        for schema, filename in test_bigpicture_xml_files:
+            gather_items.append(crud_works(schema, filename, submission_bigpicture))
```

Comment:
Good to have this

Reply:
Yea it seems `test_bigpicture_xml_files` were not being used in anything despite `conf.py` having it